Makefile

```diff
@@ -12,7 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-VERSION = 1.0.0-beta.5
+MINISHIFT_VERSION = 1.0.0-beta.5
```

Comment:
yeah, we should have done this earlier. Basically at them moment we started to have more than one version. Really, we would also in the Go code rename version to minishiftVersion, but I left that for another time.